app/src/main/resources/queries/rectangle_query_for_nearby.rq

```diff
@@ -1,16 +1,9 @@
 SELECT
   ?item
   (SAMPLE(?location) as ?location)
-  (SAMPLE(?label) AS ?label)
-  (SAMPLE(?description) AS ?description)
   (SAMPLE(?class) AS ?class)
```

Comment:
Could you please move this to the second query?

Reply:
For showing the markers on the map we will require class, endtime and destroyed. As class is used for the filtering type such as building, church  etc. Which we utilize to show different icons for the pins. And endtime, destroyed are required for the filters such as Exists.

Reply:
I believe these properties make the query slower (the query took about 10 seconds when I tried now).
Filtering is not as important as showing the first points as fast as possible.

As stated in the issue, the first query must really be only for coordinates.

Then as the points' details are loaded little by little, the points that should not be shown will be removed from the map according to the filters such as `Needs Photo` (which currently does not have any effect).

While doing tests, I recommend using FakeGPS or similar to put your location in a busy place such as the South of Manhattan. :-)

Reply:
Oh, I got the idea!